apps/web/pages/settings/my-account/appearance.tsx

```diff
@@ -90,6 +97,9 @@ const AppearanceView = () => {
     <Form
       form={formMethods}
       handleSubmit={(values) => {
+        const layoutError = validateBookerLayouts(values.defaultBookerLayouts || null);
+        if (layoutError) throw new Error(t(layoutError));
```

Comment:
This wont hit the above onError in the toast right? 

This will actually hit our error boundary? Is this intended? 

-- 
Can we just do this validation on the backend and throw the error in the toast like we do above?

Reply:
It get's caught in our default form component, which then shows that as a toast. See: 

![CleanShot 2023-05-30 at 15 54 51@2x](https://github.com/calcom/cal.com/assets/2969573/52478a09-2ad7-4295-9ab7-695435460638)

I did however find an open issue now that seems to not reset the form `isSubmitting` state to false when an error occurred. And that boolean is used in some forms to put the submit button in a loading/disabled state. So I think it makes sense to remove the client side validation here, to prevent us needing to overhaul the whole default form logic.